captum/attr/_core/occlusion.py

```diff
@@ -330,9 +351,13 @@ def _occlusion_mask(
             val for pair in zip(remaining_padding, current_index) for val in pair
         ]
         pad_values.reverse()
-        padded_tensor = torch.nn.functional.pad(sliding_window_tsr, tuple(pad_values))
+        padded_tensor = torch.nn.functional.pad(
+            sliding_window_tsr, tuple(pad_values)  # type: ignore
+        )
         return padded_tensor.reshape((1,) + padded_tensor.shape)
 
-    def _get_feature_range_and_mask(self, input, input_mask, **kwargs):
+    def _get_feature_range_and_mask(
+        self, input: Tensor, input_mask: Tensor, **kwargs: Any
+    ) -> Tuple[int, float, None]:
```

Comment:
The second return type here should also be an int. You might need to use cast below if you have errors.